selectors/Overview.bs

```diff
@@ -2031,6 +2036,49 @@ The Input Focus Pseudo-class: '':focus''</h3>
 	the same ways must apply to elements matching '':focus'' as well,
 	except that the parent of an element that matches '':focus'' must not match '':focus''.
 
+<h3 id="the-focusring-pseudo">
+The Input Focusring Pseudo-class: '':focusring''</h3>
+
+	The <dfn id="focusring-pseudo'>:focusring</dfn> pseudo-class is similar to the
```

Comment:
The mixed use of single and double quotes around the value of `id` might be a problem